tf/env/staging/secrets-gcs.tf

```diff
@@ -22,3 +22,24 @@ resource "kubernetes_secret" "gcs-hmac-key" {
     "secret-key" = google_storage_hmac_key.dev-backup-upload-key.secret
   }
 }
+
+resource "google_storage_hmac_key" "public-assets-key" {
```

Comment:
Creating an additional key is not a hard requirement to fix the issue, but it's probably a good idea in terms of helping us to debug ownership problems.